packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -167,6 +167,14 @@ export const PlatformUrls: TPlatformUrls = {
     },
 };
 
+export const Jurisdiction = {
+    BVI: 'bvi',
+    LABUAN: 'labuan',
+    MALTAINVEST: 'maltainvest',
+    SVG: 'svg',
+    VANUATU: 'vanuatu',
+} as const;
+
```

Comment:
This is duplicated in the CFDCompareAccounts/constants
It is good approach to have it in outer scope of folders.
Should i change the CFDCompareAccounts/constant value to this constant file
@mahdiyeh-deriv

Reply:
@habib-deriv remove duplicated Enums/const from the inner folder and use the outer one(
`packages/tradershub/src/features/cfd/constants.tsx`) only the things that are using just in CopmareAcoounts component should be in it's specific folder